tests/test_youtube_chapters_demo.py

```diff
@@ -0,0 +1,51 @@
+
+
+
+import os
+import sys
+import unittest
+import opentimelineio as otio
+
+
+class YoutubeChaptersDemoTests(unittest.TestCase):
+
+    def test_download_youtube_files(self):
+        youtube_url = "pvkTC2xIbeY"
+        video_file = "tmp/" + youtube_url + ".mp4"
+        description_file = "tmp/" + youtube_url + ".description"
+
+
+        # run the youtube_chapters_demo example...
+        os.system("python3 ../src/youtube_chapters_demo.py %s" % youtube_url)
```

Comment:
one nitpick -- the hashbang on the file is just `python` (not `python3`) -- you might want to run these things for both.

Also, if you expose a function in `youtube_chapters_demo()` that takes the youtube_url as an argument, you don't have to shell out.  You can call the function directly, which will give you nicer tracebacks and is generally good practice (if you don't _need_ to shell out, I usually try to avoid it).